libraries/ssh_crypto.rb

```diff
@@ -178,6 +178,7 @@ def valid_privseparation
     # define privilege separation set
     ps53 = 'yes'
     ps59 = 'sandbox'
+    ps75 = nil
```

Comment:
ps is nil by default, so I think we do not need this here

Reply:
`ps` is set to `ps59` at the line right below this. Letting `nil` be standard behaviour would let through misconfigured `sshd_config`'s with version < 7.5, right?

Reply:
@radhus You're right, good call. Missed that!